pkg/kube/inject/template.go

```diff
@@ -309,9 +309,6 @@ func cleanProxyConfig(msg proto.Message) proto.Message {
 	if proto.Equal(pc.TerminationDrainDuration, defaults.TerminationDrainDuration) {
 		pc.TerminationDrainDuration = nil
 	}
-	if proto.Equal(pc.ParentShutdownDuration, defaults.ParentShutdownDuration) {
```

Comment:
I think we can only do this if we also remove it from the proto, otherwise we should keep it

Reply:
I am planning to remove it from proto after this change is merged